app-utils/arch-test/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.22
+SRCS="git::commit=tags/v$VER::https://github.com/kilobyte/arch-test"
+CHKSUMS="SKIP"
+CHKUPDATE="gitlab::repo=kilobyte/arch-test;pattern=v\d+\.\d+;sort_version=true"
```

Comment:
It is recommended to use Anitya as the update checker:

`anitya::id=301834`

See: [release-monitoring.org/project/301834](https://release-monitoring.org/project/301834/)  
and [AOSC Find Update Syntax](https://wiki.aosc.io/developer/packaging/aosc-findupdate/)